programs/vote/src/vote_state.rs

```diff
@@ -128,6 +128,7 @@ pub struct VoteState {
 
     /// most recent timestamp submitted with a vote
     pub last_timestamp: BlockTimestamp,
+    pub has_been_slashed: bool,
```

Comment:
can you put this in epoch_credits instead of in the top level state?

Reply:
@rob-solana Is that a better spot?  I need to scan it and disable it form consensus.

Reply:
is the account disabled from consensus forever?

Reply:
yea, I think so